CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Unreleased
+
+## v0.17.1
+
+BUG FIXES:
+* Add nil check for response when unassigning roles [[GH-191]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/191)
```

Comment:
Looks like my comment got lost somehow. The suggestion I had meant to leave was to phrase this more directly in terms of impact users will see, e.g. "Avoid panic when unassigning roles", but it's very minor and not worth opening up another PR for 👍 I only wanted my review comment to make sense :-)